libs/csm/extensions/csm_prov_ext.py

```diff
@@ -0,0 +1,111 @@
+#Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""Test library for load testing related operations."""
+import time
+import logging
+from commons.utils import config_utils
+from commons.constants import  K8S_SCRIPTS_PATH, K8S_PRE_DISK
+from commons.constants import LOCAL_SOLUTION_PATH
+from libs.prov.prov_k8s_cortx_deploy import ProvDeployK8sCortxLib
+from libs.ha.ha_common_libs_k8s import HAK8s
+
+class ProvExt():
+    """RestIamUser contains all the Rest API calls for iam user operations"""
+    def __init__(self, master, workers):
+        self.master = master
+        self.workers = workers
+        self.log = logging.getLogger(__name__)
+        self.deploy_lc_obj = ProvDeployK8sCortxLib()
+        self.update_seconds = 60
+        self.ha_obj = HAK8s()
+        
+    def copy_sol_file_local(self):
+        self.log.info("Copy solution.yaml from %s to local path %s", K8S_SCRIPTS_PATH, LOCAL_SOLUTION_PATH)
+        remote_sol_path = K8S_SCRIPTS_PATH + "solution.yaml"
+        solution_path = self.master.copy_file_to_local(remote_path=remote_sol_path,local_path=LOCAL_SOLUTION_PATH)
+        return solution_path
```

Comment:
This function already exist. Check copy_sol_file in libs.prov.prov_k8s_cortx_deploy.ProvDeployK8sCortxLib

Reply:
this func copies local file to remote and not vice versa. Correct me if m wrong